nacc/logger.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import sys
+from report_handler.report_handler import ReportHandler
+
+global report_handler
```

Comment:
The report handler is not being used as intended, it should not be a global variable. Define it outside of `configure_logging` and pass it in as an argument instead. Update `configure_logging` to take in an optional `List` of `logging.Handler`s i.e., `configure_logging(config, handlers: List[logging.Handlers])` and add them to the root logger. Then each entry point script should look something like:
```python
from nacc.logging import configure_logging
from report_handler.report_handler import ReportHandler

def entrypoint():
    report_handler = ReportHandler()
    configure_logging(config, [report_handler])
   
   # run nacculator logic
   # ...

   # after nacc finishes
   report_handler.write_report()
```

Reply:
@mbentz-uf 
I have added
```python
    report_handler = ReportHandler()
    configure_logging(config, [report_handler])
```
to  the main function at
> https://github.com/ctsit/nacculator/blob/master/nacc/redcap2nacc.py (which has a function "main" at the bottom).